### DISCUSSION TITLE
Allow log in with email... and access on mutliple device

### DISCUSSION THREAD

**Discussion by myckaa** [iSoron/uhabits#1330] [Feature Requests] [answered]
Hi ! 
First, i LOVE the apps, giving for me all of what i need!!!! 
The only thing is missing to be perfect, is to be able to log-in option... it would be VERY usefull to be able to track my habits on a different device with Android.. 

With that, the application will be complete and so perfect! Do you think it could be a possible add ?

Thank!

**Comment by LordJon** [comment, marked as answer]
There's currently a synchronisation function being developed. It was planned for version 2.0 but needed polishing so they postponed it to version 2.1 (correct me if I'm wrong but I think I recently saw it being scheduled further ahead to version 2.2).

Edit: here's the original request: https://github.com/iSoron/uhabits/discussions/104
